src/components/text-editor/prosemirror-adapter/plugins/trigger/factory-helpers/text-input-handler.spec.ts

```
@@ -2,7 +2,7 @@ jest.mock('./send-trigger-event', () => ({
     sendTriggerEvent: jest.fn(),
 }));
 
-import { TriggerCharacter } from 'src/interface';
+import { TriggerCharacter } from '@limetech/lime-elements'
```

ESLint

[error] Insert `;`

(prettier/prettier)